index.js

```diff
@@ -258,6 +262,10 @@ if ((os.platform() === 'win32' && require('is-administrator')()) || process.getu
             return macaddress.all(cb);
         });
     }], (error, all) => {
+        if(error) {
```

Comment:
spacing here, do we have lint rules turned on for this repo?
(space after `if`)

Reply:
@mikehu, we don't have it in testing, it seems, but we have `.eslintrc` inside the repo - @vilyapilya, can you do `eslint --fix -c .eslintrc index.js` and push the changes? Thanks in advance!